cpp/tests/t/pipelines/slac/SLAC.cpp

```diff
@@ -71,9 +71,8 @@ static bool IsPointCloudSimilar(t::geometry::PointCloud source,
 TEST_P(SLACPermuteDevices, DISABLED_RunSLACOptimizerForFragments) {
     core::Device device = GetParam();
 
-    std::string dataset_folder =
-            std::string(TEST_DATA_DIR) +
-            "/reconstruction_system/livingroom1_clean_micro";
+    std::string dataset_folder = utility::GetDataPathCommon(
+            "/reconstruction_system/livingroom1_clean_micro");
```

Comment:
we may remove the first slash:
`/reconstruction_system/livingroom1_clean_micro` -> `reconstruction_system/livingroom1_clean_micro`.

Do we want to handle this automatically in the `GetDataPathCommon` (like join(path1, path2) in python?

Reply:
Done.

We recommend removing the `/` at the beginning. But both will work since `//` or even `///` will be interpreted as `/` eventually by the system.